src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/services/TestCaseDisplayService.kt

```diff
@@ -25,13 +28,25 @@ class TestCaseDisplayService(private val project: Project) {
     private var editorList: MutableList<Pair<String, EditorTextField>> = arrayListOf()
     private val highlightColor: Color = Color(100, 150, 20, 30)
 
+    // Variable to keep reference to the coverage visualisation content
+    private var content: Content? = null
+
     init {
         allTestCasePanel.layout = BoxLayout(allTestCasePanel, BoxLayout.Y_AXIS)
         mainPanel.layout = BorderLayout()
         mainPanel.add(applyButton, BorderLayout.SOUTH)
         mainPanel.add(scrollPane, BorderLayout.CENTER)
     }
 
+    /**
+     * Creates the complete panel in the "Generated Tests" tab,
+     * and adds the "Generated Tests" tab to the sidebar tool window.
+     */
+    fun showGeneratedTests(testReport: CompactReport) {
+        displayTestCases(testReport)
```

Comment:
Should I make displayTestCases() function private, since it is now only used inside this function?

Reply:
That would indeed make sense.